lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt

```diff
@@ -90,6 +96,12 @@ class LapisController(
         )
         @RequestParam
         offset: Int? = null,
+        @Parameter(
+            schema = Schema(ref = "#/components/schemas/$FORMAT_SCHEMA"),
+            description = "The format of the response.",
```

Comment:
Let's maybe reuse the description that we already have in the schema in `OpenApiDocs`? Maybe we could add there that the parameter takes precedence over the header.